internal/transport/http2_client_test.go

```diff
@@ -27,6 +27,11 @@ import (
 	"google.golang.org/grpc/resolver"
 )
 
+const IntendedTestCaseString = "This is a test case string"
```

Comment:
Why are defining your own string? Is this the expected string for which you will force error? If yes, name it exactly as the passed string testcase. For example, if it is for `framerWriteSettings`, just name it as `framerWriteSettingsString`. 

Also, don't you need 4 such strings i.e. one per test case? Right now you only have 2

Reply:
change done